go/vt/vtgate/endtoend/vstream_test.go

```diff
@@ -279,6 +279,11 @@ func TestVStreamCopyUnspecifiedShardGtid(t *testing.T) {
 		require.NoError(t, err)
 	}
 
+	_, err = conn.ExecuteFetch("insert into t1_copy_all_ks3(id1,id2) values(30,30), (40,40)", 1, false)
+	if err != nil {
```

Comment:
I know other tests are already doing it, but I don't think this is the way `require.NoError(t, err)` is meant to be used. no need to check `err` for nil first. would you mind fixing the file or at least method while you are here? 🙇

Reply:
hmm... you marked this as resolved, but the method/file still check `err` for nil before `require.NoError(t, err)`

Reply:
ah i had reverted all the changes to the file after moving benchmarks to `go/test/endtoend/vtgate`, so this escaped my. fixed in 8bbae3f442.